examples/examples/z_sub.rs

```diff
@@ -39,23 +36,13 @@ async fn main() {
 
     let subscriber = session.declare_subscriber(&key_expr).res().await.unwrap();
 
-    println!("Enter 'q' to quit...");
-    let mut stdin = async_std::io::stdin();
-    let mut input = [0_u8];
+    println!("Press CTRL-C to quit...");
     loop {
         select!(
```

Comment:
Select is no longer necessary, just have a `while let Ok(sample) = subscriber.recv().await { .. }`.